hazelcast/src/main/java/com/hazelcast/cp/internal/RaftService.java

```diff
@@ -642,6 +658,11 @@ void updateMissingMembers() {
         }
     }
 
+    private boolean skipUpdateMissingMembers() {
+        return config.getMissingCPMemberAutoRemovalSeconds() == 0 || !metadataGroupManager.isDiscoveryCompleted()
```

Comment:
nit: I'd move the || part to the next line, will be a bit clearer to see that there are 3 cases, one with compound expression